crossbar/controller/process.py

```diff
@@ -346,7 +347,7 @@ def _start_native_worker(self, wtype, id, options=None, details=None):
 
         # all native workers (routers and containers for now) start from the same script
         #
-        filename = pkg_resources.resource_filename('crossbar', 'worker/process.py')
+        filename = FilePath(crossbar.__file__).parent().child("worker").child("process.py").path
 
```

Comment:
Sorry, you'll have to rephrase. The `FilePath` usage here is because os.path has no `parent()` equiv, and using `__file__` rather than pkg_resources means that we use the one in the source tree that Crossbar is being ran from -- so if you've got it installed, it won't inadvertently use the installed copy (which is why I had to change it, it was using the one in system packages).